src/index.html

```diff
@@ -9,6 +9,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="/" class="nav_logo">
+        <img src="./images/logo.png" alt="MOYO">
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link is-active" href="#apple">Apple</a>
+          </li>
+        </ul>
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a class="nav_link" href="#samsung">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav_list">
          <li class="nav_item">
            <a class="nav_link is-active" href="#apple">Apple</a>
          </li>
          <li class="nav_item">
            <a class="nav_link" href="#samsung">Samsung</a>
          </li>
          ...
       </ul>
```
should be only one `ul` tag with list items (`li`) inside